app/models/referee_interface/reference_form.rb

```diff
@@ -0,0 +1,24 @@
+module RefereeInterface
+  class ReferenceForm
```

Comment:
We should use (a version of / refactored version of) `ReceiveReference` instead of this new form object. 

In the flow, this makes sure that when all references are complete the state is changed and we send the applications to the providers.

Reply:
As we discussed, it's fine to have a bit of duplication.